app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import { Table as NeetoUITable, Pagination } from "neetoui";
+
+import { buildContactsColumnData } from "./utils";
+
+import { CONTACT_DETAILS } from "../constants";
+
+const Table = () => (
+  <>
+    <NeetoUITable
```

Comment:
Why do we need this fragment here?

Reply:
There is a pagination component as well, that's why I added the React Fragment.

Reply:
Got it. Didn't see the Pagination component.